sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/InputForbidFieldAccessCheckStaticTest.java

```diff
@@ -0,0 +1,18 @@
+package com.github.sevntu.checkstyle.checks.coding;
+
+import com.puppycrawl.tools.checkstyle.api.LocalizedMessage;
+
+import static java.util.Locale.ROOT;
+import static java.lang.Integer.MAX_VALUE;
+
+public class InputForbidFieldAccessCheckStaticTest {
```

Comment:
Input files should not end with `Test`. It will confuse people with actual tests.